src/ui/components/FogBackground/template.hbs

```diff
@@ -1,16 +1,4 @@
 <div data-test-fog-background>
-  <div class="intentisty" style={{opacityStyle}}>
-    <div class="fog-01">
-      <div class="image-01" />
-      <div class="image-01" />
-    </div>
-    <div class="fog-02">
-      <div class="image-02" />
-      <div class="image-02" />
-    </div>
-    <div class="fog-03">
-      <div class="image-01" />
-      <div class="image-01" />
-    </div>
-  </div>
+  <canvas id='ParticlesBackgroundCanvas' />
```

Comment:
Also we should always use double quotes in HTML 🙏 (I realized this repo is lacking a good linting setup).